### ISSUE TITLE
Dataset?

### ISSUE THREAD

**Issue by jhkd-kevin** [superstarNathan/VTGAN-pytorch#1]
Hi! I also want to reproduce VTGAN, but the connection of his original data set fails. Could you share your data set with me? Thank you!

**Comment by superstarNathan** [comment]
hi, I uploaded the dataset. You can check .rar files. Later i will add them to dataset file.
By the way, I fail to reproduce VTGAN though my code, I guess that my loss functions donot work and I miss something in it, i don't know how to correct and improve them. I will appreciate it if you can give me some advise.
